packages/arb-bridge-eth/contracts/arch/OneStepProof.sol

```diff
@@ -592,9 +590,7 @@ library OneStepProof {
         returns (bool)
     {
         machine.addDataStackValue(
-            Value.newBoolean(
-                Value.hash(machine.dataStack) == Value.hashEmptyTuple()
-            )
+            Value.newBoolean(machine.dataStack.hash() == Value.newNone().hash())
```

Comment:
do all default Values have the same hash as empty tuples?

Reply:
`newNone` really should have been called `newEmptyTuple`. I've not renamed that method. I'm not sure what you mean by default Values?